### ISSUE TITLE
Inconsistent condition/action names

### ISSUE THREAD

**Issue by BWPanda** [4ian/GDevelop#846]
## Describe the bug
The conditions/actions for setting the text of a scene/global variable are named differently depending on if you're adding a condition or an action. This makes it hard to find them when searching.

## To Reproduce
1. Click 'Add condition' in the Events editor
2. Expand the Variables group on the left
3. Note the names of the conditions for setting the text of a scene and global variable ('**Text** of a [scene/global] variable')
4. Cancel, and click 'Add action'
5. Expand the Variables group on the left
6. Note the names of the conditions for setting the text of a scene and global variable ('**String** of a [scene/global] variable') (note also that clicking on one of these shows the title 'Modify the **text** of a [scene/global] variable')

I think naming these consistently would be better.

![text_vs_string](https://user-images.githubusercontent.com/2385329/50727815-f2b7b300-1128-11e9-9516-de524fd2fb8e.jpg)

**Comment by 4ian** [comment]
Right! The question is now, should we favor "string" (which is the "programming" term, also used in the expression like VariableString or ToString), or should we favor "text" (which is maybe more accessible/understandable, but not used in expressions)?

**Comment by Lizard-13** [comment]
I prefer string, string everywhere we talk about string values, not only for consistency with the expressions but also to not mix it with the text object. Text object uses "string" too :)

**Comment by 4ian** [comment]
Maybe for actions and conditions titles, we can actually write both:
"String (text)", "String (text) of a scene variable"...

Could make things both exact (string is the proper term to use everywhere) but clearer especially for beginners who don't know what a string is?

**Comment by Lizard-13** [comment]
Sounds fair :+1:

**Comment by Endoperez** [comment]
I'm afraid I don't agree with a new translation of "String (text)". 

Text is already used when string is meant on multiple different conditions and actions: Text Object, text of text object, change the text of a scene variable, check if the text of scene variable X is = "something", and so on. 

To remain consistent, either all of those would need to be changed, or at the very least, it should read "Text (string)".

**Comment by 4ian** [comment]
> To remain consistent, either all of those would need to be changed, or at the very least, it should read "Text (string)".

The idea was to change everything, not only condition/actions related to variables.

> Also, I made a thread about this on the Crowdin page, but I think the English localization could be improved in general. This isn't the only case where some updating is needed. For example "Common conditions for...", "Common action for..." (plural vs singular in the category name).

I've not seen it this. I've created this issue to report any mistake/typo/inconsistency: https://github.com/4ian/GDevelop/issues/927 - can you post here or link your thread?

Fixed "Common action for ..." to "Common actions for ..."  for next version :)

**Comment by HelgeStenstrom** [comment]
Is there anything to do on this issue? It seems to me that issue #4041 covers what is in the description of this issue.

**Comment by ClementPasteau** [comment]
closing as already handled
